packages/google_maps_flutter/google_maps_flutter_ios/example/ios14/ios/RunnerTests/GoogleMapsGroundOverlayControllerTests.m

```diff
@@ -0,0 +1,155 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+@import google_maps_flutter_ios;
+@import google_maps_flutter_ios.Test;
+@import XCTest;
+@import GoogleMaps;
+
+#import <OCMock/OCMock.h>
+#import <google_maps_flutter_ios/FGMGroundOverlayController_Test.h>
+#import <google_maps_flutter_ios/messages.g.h>
+#import "PartiallyMockedMapView.h"
+
+@interface GoogleMapsGroundOverlayControllerTests : XCTestCase
+@end
+
+@implementation GoogleMapsGroundOverlayControllerTests
+
+/// Returns GoogleMapGroundOverlayController object instantiated with position and a mocked map
+/// instance
+///
+/// @return An object of FLTGoogleMapGroundOverlayController
+- (FGMGroundOverlayController *)groundOverlayControllerWithPositionWithMockedMap {
```

Comment:
Do these need to be instance methods rather than class methods?

Reply:
Actually no. Changed them to class methods.